krabs/krabs/ut.hpp

```diff
@@ -137,11 +147,10 @@ namespace krabs { namespace details {
             parameters.SourceId = provider.first;
             
             GUID guid = provider.first;
-            parameters.EnableProperty = std::get<3>(provider.second.flags_tuple_);
+            parameters.EnableProperty = provider.second.filter_flags_.trace_flags_;
             parameters.EnableFilterDesc = nullptr;
             parameters.FilterDescCount = 0;
             EVENT_FILTER_DESCRIPTOR filterDesc;
```

Comment:
`EVENT_FILTER_DESCRIPTOR filterDesc{};` to agg-init to zero and then you don't need the `ZeroMemory` call below.